# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/style/FontFaceSetImpl.cpp
# Commit: 81cd4125a2d2
# Full Hash: 81cd4125a2d2f6db7d17addb9e93b8c61dde1f66
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2022-07-12 04:20:08
# Description:
#   Bug 1779096 - Clear FontFaceSetImpl::mOwner when destroyed. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D151554
# ==============================================================================

diff -r bfa523daaf13 -r 81cd4125a2d2 layout/style/FontFaceSetImpl.cpp
--- a/layout/style/FontFaceSetImpl.cpp	Mon Jul 11 23:23:53 2022 +0000
+++ b/layout/style/FontFaceSetImpl.cpp	Mon Jul 11 23:24:36 2022 +0000
@@ -93,10 +93,9 @@
   }
 
   mLoaders.Clear();
-
   mNonRuleFaces.Clear();
-
   gfxUserFontSet::Destroy();
+  mOwner = nullptr;
 }
 
 void FontFaceSetImpl::ParseFontShorthandForMatching(
